wondrous-app/components/BreadCrumbs/Components/PodSelector.tsx

```diff
@@ -0,0 +1,133 @@
+
+  const handleClick = (event) => setAnchorEl(anchorEl ? null : event.currentTarget);
+  const handleClickAway = () => setAnchorEl(null);
+
+  const open = Boolean(anchorEl);
+
+  const [getUserPods, { data, loading, error }] = useLazyQuery(GET_USER_AVAILABLE_PODS, {
+    fetchPolicy: 'cache-and-network',
+    nextFetchPolicy: 'cache-first',
+    notifyOnNetworkStatusChange: true,
+  });
+  const selectedValue = data?.getAvailableUserPods?.find((option) => option.id === id);
+
+  useEffect(() => {
+    if (open && !data) {
+      getUserPods({
+        variables: {
+          orgId: activePod?.orgId,
+        },
+      });
+    }
+  }, [open]);
+
+  return (
+    <PodSearchClickAway onClickAway={handleClickAway}>
+      <div>
+        <PodSelectorWrapper onClick={handleClick} display="flex" justifyContent="center" alignItems="center" gap="8px">
+          <PodIcon
+            color={color}
+            style={{
+              width: 24,
+              height: 24,
+              borderRadius: 50,
+            }}
+          />
+          <PodTitle>{name}</PodTitle>
+          <ArrowIcon open={open} />
+        </PodSelectorWrapper>
+        <PodSearchPopper open={open} anchorEl={anchorEl} placement="bottom-start" disablePortal>
+          {loading ? (
+            <Spinner />
+          ) : (
+            <PodSearchAutocomplete
+              value={selectedValue}
+              renderInput={(params) => (
+                <PodSearchInput
+                  {...params}
+                  ref={params.InputProps.ref}
+                  disableUnderline
+                  fullWidth
+                  autoFocus
+                  InputProps={{
+                    ...params.InputProps,
+                    endAdornment: (
+                      <PodSearchInputAdornment position="end">
+                        <PodSearchInputIcon />
+                      </PodSearchInputAdornment>
+                    ),
+                  }}
+                />
+              )}
+              disableClearable
+              isOptionEqualToValue={(option, value) => option.id === value?.id}
+              getOptionLabel={(option) => option.name}
+              renderOption={(props, option) => (
+                <PodSearchListItem {...props} isActive={option.id === selectedValue?.id}>
+                  <PodSearchDefaultImage color={option?.color ?? '#474747'} />
+                  <PodSearchLabel>{option?.name}</PodSearchLabel>
+                </PodSearchListItem>
+              )}
+              PaperComponent={PodSearchPaper}
+              ListboxComponent={PodSearchList}
+              PopperComponent={(params) => <PodSearchAutocompletePopper {...params} />}
+              open={open}
+              options={data?.getAvailableUserPods ?? []}
+              disablePortal
+              onChange={(event, value, reason) => {
+                if (reason === 'selectOption') {
+                  const searchParams = new URLSearchParams(window.location.search);
```

Comment:
Any reason we're using `window.location` instead of `router`?

Reply:
changing entityType on boards page doesn't trigger a router state update, only changes the url in case the user wants to copy the url with the entityType inside, the logic behind this was to basically  keep the entityType on pod change, so if the user is in pod / podId / bounties , on podId change keep the user on the bounties. But on a second thought looks hackish, changed it now while keeping the same functionality

Reply:
Gotcha makes sense!